src/main/java/org/tron/core/db/Manager.java

```diff
@@ -107,6 +108,9 @@
 @Slf4j
 @Component
 public class Manager {
+
+  public String resource = "FullNode";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make resource a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1609&issues=AWZ_YhHQP8lmccBDZe2_&open=AWZ_YhHQP8lmccBDZe2_)

[ISSUE_KEY]: AWZ_YhHQP8lmccBDZe2_